### PR TITLE
fix standard quota issue, update quota id

### PR DESCRIPTION
**Pull request by ajaypratap003**
https://issues.redhat.com/browse/MGDSTRM-7932

### CODE REVIEW

**Comment by wtrocki** [comment]
This fixes current issue however in long run we should not rely on those values as they care just identifiers that aren't considered an unique identifiers.
Screenshot 2022-03-24 at 15 35 08

**Comment by ziccardi** [comment]
@wtrocki not just the product, but the couple product and resource_name.

standard instance: resource_name='rhosak' and product='RHOSAK'
eval instance: resource_name='rhosak' and product='RHOSAKTrial'

**Comment by ajaypratap003** [comment]
@ziccardi @wtrocki Thanks for the suggestions. I've added check for product and  resource_name. Please have a look into this PR.

File: src/app/hooks/quota.ts [hunk 1]
```diff
@@ -30,39 +30,42 @@ export const useQuota = (productId: ProductType) => {
...
             } as Configuration);
-
             await ams
                 .apiAccountsMgmtV1OrganizationsOrgIdQuotaCostGet(orgId, undefined, true)
                 .then((res) => {
                     const quotaData = new Map<QuotaType, QuotaValue>();
                     const quota = res?.data?.items?.filter(
-                        (q) => q.quota_id.trim() === quotaId
+                        (q) => q.related_resources?.filter((r) => r.resource_name === resourceName && r.product === quotaProductId)[0]
                     )[0];
```
Comment (ziccardi) [inline src/app/hooks/quota.ts:65]: We get just the first one: why not using `find` instead of `filter`?

File: src/app/hooks/quota.ts [hunk 2]
```diff
@@ -30,39 +30,42 @@ export const useQuota = (productId: ProductType) => {
...
 
             const accessToken = await auth?.ams.getToken();
             const ams = new AppServicesApi({
                 accessToken,
                 basePath: config?.ams.apiBasePath || '',
             } as Configuration);
-
             await ams
                 .apiAccountsMgmtV1OrganizationsOrgIdQuotaCostGet(orgId, undefined, true)
                 .then((res) => {
```
Comment (ziccardi) [inline src/app/hooks/quota.ts:61]: await ams...then: is that correct?

Comment (ajaypratap003) [reply src/app/hooks/quota.ts:62]: yes, It's working fine. Here we have mix of await and then. I'll update it. Thanks for the suggestion.

Comment (ziccardi) [reply src/app/hooks/quota.ts]: yeah, it works, but it converts the `non async` method to e `resolved promise`, causing some overhead. See [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await#description)

**Pull request merged**